editor.planx.uk/src/pages/FlowEditor/components/Settings/GeneralSettings/ContactForm.tsx

```diff
@@ -0,0 +1,83 @@
+import { useFormik } from "formik";
+import React, { ChangeEvent } from "react";
+import InputDescription from "ui/editor/InputDescription";
+import Input from "ui/shared/Input";
+import InputRow from "ui/shared/InputRow";
+import InputRowLabel from "ui/shared/InputRowLabel";
+import * as Yup from "yup";
+
+import { SettingsForm } from "../shared/SettingsForm";
+import { FormProps } from ".";
+
+export default function ContactForm({ formikConfig, onSuccess }: FormProps) {
+  const formSchema = Yup.object().shape({
+    helpEmail: Yup.string()
+      .email("Please enter valid email")
+      .required("Help Email is required"),
+    helpPhone: Yup.string().required("Help Phone is required"),
+    helpOpeningHours: Yup.string(),
+  });
```

Comment:
Perfect!

Reply:
@DafyddLlyr maybe this is something we pick up with @ianjon3s as well, but in terms of UX copy and tone of voice in message handling, is there a particular way to write these error messages? 

To ensure consistency in tone, I know people do like to manage how these a written

Reply:
We don't have standard copy for Editor-facing errors, but generally try to get input from the content team (+ Nomensa URs) on public-facing error messages.

I'm pretty sure most Editor-facing inputs just currently use `.required()` and would output whatever Yup outputs which would be something like `${fieldName} is required`

Reply:
@DafyddLlyr @ianjon3s worth noting the advice here: https://opensystemslab.notion.site/How-to-write-good-planning-service-content-30313a55fc5b4499846e32e88b1cf76a#b02887d8bd1d4cbe8e4d6d842a3a398b